romeo/src/event.rs

```diff
@@ -0,0 +1,28 @@
+use bdk::bitcoin::{Block, Txid as BitcoinTxId};
+use blockstack_lib::burnchains::Txid as StacksTxId;
+
+use crate::state::DepositInfo;
+use crate::state::WithdrawalInfo;
+
+/// Events are spawned from tasks and used
+/// to update the system state.
+#[derive(Debug, Clone)]
+pub enum Event {
+    MintCreated(DepositInfo, StacksTxId),
+    BurnCreated(WithdrawalInfo, StacksTxId),
+    FulfillCreated(WithdrawalInfo, BitcoinTxId),
```

Comment:
In this case I don't think it is. I think it's clear enough from the context that the transaction ID here refers to the created fulfill transaction and not the withdrawal transaction - since it is the same pattern with the previous calls as well.